src/main/resources/data/bloodmagic/patchouli_books/guide/ru_ru/templates/crafting_altar.json

```diff
@@ -0,0 +1,52 @@
+{
+  "processor": "wayoftime.bloodmagic.compat.patchouli.processors.BloodAltarProcessor",
+  "components": [
+    {
+      "type": "header",
+      "text": "#heading",
+      "x": -1,
+      "y": -6
+    },
+    {
+      "type": "image",
+      "image": "bloodmagic:textures/gui/patchouli_book/crafting.png",
+      "x": 9,
+      "y": 4,
+      "u": 0,
+      "v": 51,
+      "texture_width": 128,
+      "texture_height": 256,
+      "width": 98,
+      "height": 60
+    },
+    {
+      "type": "item",
+      "item": "bloodmagic:altar",
+      "x": 87,
+      "y": 46
+    },
+    {
+      "type": "item",
+      "item": "#input",
+      "x": 30,
+      "y": 8
+    },
+    {
+      "type": "item",
+      "item": "#output",
+      "x": 87,
+      "y": 27
+    },
+    {
+      "type": "text",
+      "text": "Tier: #tier# $(br)LP: #lp#",

```

Comment:
This text shows on all Crafting Altar recipes.  Need to translate "Tier:" and "LP" (Life Points).